Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -0,0 +1,1325 @@
+from typing import Union, Dict, Optional, Any
+
+import demistomock as demisto
+import requests
+from CommonServerPython import *  # noqa: E402 lgtm [py/polluting-import]
+from CommonServerUserPython import *  # noqa: E402 lgtm [py/polluting-import]
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+# GLOBALS
+CB_ORG_KEY: str
+
+
+def convert_unix_to_timestamp(timestamp):
+    """
+    Convert millise since epoch to date formatted MM/DD/YYYYTHH:MI:SS
+    """
+    if timestamp:
+        date_time = datetime.utcfromtimestamp(timestamp / 1000)
+        return date_time.strftime('%Y-%m-%dT%H:%M:%SZ')
+    return ''
+
+
+def convert_date_to_unix(dstr):
+    """
+    Convert a given string with MM/DD/YYYY format to millis since epoch
+    """
+    dt = datetime.strptime(dstr, '%Y-%m-%dT%H:%M:%SZ')
+    return dt.timestamp()
+
+
+class Client(BaseClient):
+    """
+    Client will implement the service API, and should not contain any Demisto logic.
+    Should only do requests and return data.
+    """
+    def __init__(self, base_url: str, use_ssl: bool, use_proxy: bool, token=None):
+        self.token = token
+        super().__init__(base_url, verify=use_ssl, proxy=use_proxy, headers={'Accept': 'application/json',
+                                                                             'Content-Type': 'application/json'})
+        if self.token:
```

Comment:
do this before calling to `BaseClient` constructor, and initalize the headers once, no need initialize and then update